certified-connectors/ProcessStreet/apiDefinition.swagger.json

```diff
@@ -518,46 +475,55 @@
         } ],
         "responses" : {
           "200" : {
-            "description" : "The result of the form fields update.",
+            "description" : "The updated workflow run form fields",
             "schema" : {
               "$ref" : "#/definitions/UpdateMultipleFormFieldValuesResponse"
             }
-          },
-          "400" : {
-            "description" : "Invalid value for: body",
+          }
+        }
+      }
+    },
+    "/users" : {
+      "get" : {
+        "summary" : "Get users",
+        "description" : "Retrieves all the users",
+        "operationId" : "ListUsers",
+        "consumes" : [ "application/json" ],
+        "produces" : [ "application/json" ],
+        "parameters" : [ {
```

Comment:
@mwysokinski please fix the formatting here so that the object starts on a new line.

Reply:
fixed